opil/opil_factory.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, uri):
                 elif datatypes[0] == 'http://www.w3.org/2001/XMLSchema#integer':
                     self.__dict__[property_name] = sbol.IntProperty(self, property_uri, 0, upper_bound)                    
                 elif datatypes[0] == 'http://www.w3.org/2001/XMLSchema#boolean':
-                    self.__dict__[property_name] = sbol.BooleanProperty(self, property_uri, 0, 1)
```

Comment:
We have a need for instances to have multiple values of a string property (this is actually pretty common, for example things can have multiple names) but it doesn't really make sense to have multiple values for boolean and integer properties.

Reply:
Understood. The code is only written this way for generality.  In practice, none of the OPIL boolean and integer properties support multiple values, as you noted.  I just wrote the autogeneration code to be general, in case this ever changes, or if we want to port the autogeneration code to other ontologies.